modules/external_commands/inc/com/centreon/engine/modules/external_commands/processing.hh

```diff
@@ -249,17 +307,18 @@ namespace         modules {
 
         char* group_name(my_strtok(args, ";"));
         servicegroup_map::const_iterator sg_it{servicegroup::servicegroups.find(group_name)};
-        if (sg_it == servicegroup::servicegroups.end())
+        if (sg_it == servicegroup::servicegroups.end() ||
+          sg_it->second.get() == nullptr)
```

Comment:
boolean operator works on shared_ptr, `sg_it->second.get() == nullptr` is the same as `!sg_it->second`. :-p

Reply:
done